src/client/pages/Landing/Landing.tsx

```diff
@@ -13,31 +13,113 @@ import {
 import { Page } from 'components/utils/Page'
 import { useVariation, Variation } from 'utils/hooks/useVariation'
 import { TextKeyMap, useTextKeys } from 'utils/textKeys'
+import { CheckmarkCircle } from 'components/icons/CheckmarkCircle'
 import { LanguagePicker } from '../Embark/LanguagePicker'
 import { Card } from './components/Card'
 
 const Wrapper = styled.div`
-  display: flex;
+  display: block;
   width: 100%;
   max-width: 80rem;
   margin: 0 auto;
-  align-items: center;
   padding: 1rem 1rem 4rem;
 
   @media (min-width: 600px) {
-    padding-top: 10vh;
+    padding-top: 6vh;
   }
 
-  @media (min-width: 850px) {
-    padding-top: 25vh;
+  @media (min-height: 900px) and (min-width: 1020px) {
+    padding-top: calc(25vh - 10.5rem);
+  }
+`
+
+const UspContainer = styled.div`
+  display: block;
+  margin-bottom: 1.5rem;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  color: ${colorsV3.gray100};
+
+  @media (min-width: 800px) {
+    margin-bottom: 2rem;
+    text-align: center;
+  }
+
+  @media (min-width: 1020px) {
+    margin-bottom: 4rem;
+  }
+`
+const Headline = styled.h1`
+  margin-top: 0;
+  margin-bottom: 1.25rem;
+  font-size: 2rem;
+  letter-spacing: -0.02em;
+
+  @media (min-width: 800px) {
+    margin-bottom: 0.75rem;
+  }
+
+  @media (min-width: 1020px) {
+    font-size: 3rem;
+    line-height: 3.5rem;
+  }
+`
+
+const Preamble = styled.p`
+  margin-top: 0;
+  margin-bottom: 1.25rem;
+  line-height: 1.5;
+
+  @media (min-width: 800px) {
+    margin-bottom: 2rem;
+    font-size: 1.125rem;
+  }
+
+  @media (min-width: 1020px) {
+    margin-bottom: 2rem;
+    font-size: 1.5rem;
+  }
+`
+
+const UspList = styled.ul`
+  margin: 0;
+  padding-left: 0;
+  list-style: none;
+  text-align: left;
+
+  @media (min-width: 800px) {
+    display: inline-flex;
+  }
+`
+
+const UspItem = styled.li`
+  display: flex;
+  margin-bottom: 0.5rem;
+  align-items: center;
+
+  span {
+    margin-left: 1rem;
+    line-height: 1.5;
+  }
+
+  @media (min-width: 800px) {
+    display: inline-flex;
+
+    span {
+      margin-right: 2rem;
+    }
+  }
+
+  @media (min-width: 1020px) {
+    span {
+      margin-right: 3rem;
+    }
   }
 `
 
-const Container = styled.div`
+const CardContainer = styled.div`
```

Comment:
this is nit, and I see this was the case before also but should we decide on either "container" or "wrapper" and not mix the two? Imo wrapper is better to not confuse it with a container of state/functionality?

Reply:
I get your point :) Personally I'm not a fan of Wrapper and I prefer Container (as the name for a wrapping div for a component) and that's what I've seen and used in most projects. I see what you mean with react container components, however I don't feel that has been a source of confusion for me historically :) But I'm up for discussion!

Reply:
I actually also like the "container" more. I've seen people argue that container is more for when you have multiple things grouped and wrapper is to  wrap one or a few things. Personally I think of the wrapper as bigger than a container. Like if you need to add styling to an entire component or something. What I wanted to say with this is that I think it's fine to mix them if we do it consistently 😇

Reply:
I think, I have used wrapper as a wrapper of one single element, and container to "contain" multiple things, hence I agree with Ellen that it's okay to use both but we should be consistent with how we used them.